pkgs/instantLock/default.nix

```
@@ -1,9 +1,6 @@
-{ stdenv, lib, fetchurl, writeText
-, fetchFromGitHub
-, argtable3, glib, wrapGAppsHook, pkg-config
-, xorgproto, libX11, libXext, libXrandr, libXinerama
-, instantMenu
-, conf ? null }:
+{ stdenv, lib, fetchurl, writeText, fetchFromGitHub, argtable3, glib
+, wrapGAppsHook, pkg-config, xorgproto, libX11, libXext, libXrandr, libXinerama
+, libxcrypt, instantMenu, conf ? null }:
```

How about adapting the nixpkgs-fmt style, so that people won't have to change the format to add/remove an argument.

```suggestion
{ stdenv
, lib
, fetchurl
, writeText
, fetchFromGitHub
, argtable3
, glib
, wrapGAppsHook
, pkg-config
, xorgproto
, libX11
, libXext
, libXrandr
, libXinerama
, libxcrypt
, instantMenu
, conf ? null
}:
```